database/db_dataframe.py

```diff
@@ -0,0 +1,149 @@
+import sqlite3
```

Comment:
Write a doctype:
"""
This class contains all the indicators that produce from the database.
if we add a new indicator that will reflect the application automatically.
"""